package.json

```diff
@@ -1 +1,7 @@
-{}
+{
+  "dependencies": {
```

Comment:
These dependencies should be added to the package.json inside of "libre311/frontend" instead of the root